Mergin/plugin.py

```diff
@@ -232,6 +237,24 @@ def configure(self):
             self.on_config_changed()
             self.show_browser_panel()
 
+    def post_login(self):
+        """Groups actions that needs to be done when auth information changes"""
+        if not self.mc:
+            return
+
+        # check action required flag
+        service_response = self.mc.user_service()
+
+        if service_response and type(service_response) is dict:
+            requires_action = service_response.get("action_required", False)
+
+            if requires_action:
+                iface.messageBar().pushMessage(
+                    "Mergin",
+                    f"Your attention is required.&nbsp;Please visit the <a href={self.mc.url}>Mergin dashboard</a>",
```

Comment:
I would rather be explicit (/dashboard) in case there would be some issue with redirect.

Do we want to use UTM tags (similar to Input)?